vtex/loaders/legacy/productListingPage.ts

```diff
@@ -94,14 +94,27 @@ const getTerm = (path: string, map: string) => {
   const mapSegments = map.split(",");
   const pathSegments = removeForwardSlash(path).split("/");
 
-  return pathSegments.slice(0, mapSegments.length).join("/");
+  const term = pathSegments.slice(0, mapSegments.length).join("/");
+  console.log(mapSegments);
+  if (mapSegments.includes("priceFrom")) {
+    return term.replace(/de-\d+[,]?[\d]+-a-\d+[,]?[\d]+/, (match) => {
+      return match.replaceAll(",", ".");
+    });
+  }
```

Comment:
Can you create named variables to better explain the algorithm here?

Ex:

const `termWithoutWhatever = term.replace(/de-\d...`

(I can't know what this is doing)

Reply:
Fixed